Sources/PostgresNIO/New/PSQLData.swift

```diff
@@ -1,8 +1,12 @@
-/// The format code being used for the field.
-/// Currently will be zero (text) or one (binary).
-/// In a RowDescription returned from the statement variant of Describe,
-/// the format code is not yet known and will always be zero.
-enum PSQLFormatCode: Int16 {
+/// The format the postgres types are encoded in on the wire.
+///
+/// Currently there a two wire formats supported:
+///  - text
+///  - binary
+///
+/// In a `RowDescription` returned from the statement variant of `Describe`,
+/// the format is not yet known and will always be `.text`.
```

Comment:
I don't understand this, does it mean that we assume it to be `.text` until we know otherwise?

Reply:
Removed the sentence in question. this is explained in ExtendedQueryStateMachine much better. And is really only relevant there.

Reply:
https://github.com/vapor/postgres-nio/pull/158/files#r631609703